04_getting_data_01_csvfiles.Rmd

```diff
@@ -64,7 +64,30 @@ Above, you see the simplest way to import a CSV file. However, as with many func
 
 By default, `read_csv()` converts blank cells to missing data (NA).
 
-Finally, we introduce the function `read_csv` here and recommend that you use it, as it is the simplest and fastest way to read CSV files into R. However, we note that there *is* a function `read.csv` which is available by default in R. You will likely see this function in others' code, so we just want to make sure you're aware of it.
+Note that the `read_csv()` function is a part of a *family* of functions that all have similar names.
+
+For example, if you need to read in a **TSV** file, you can use `read_tsv()` function. If you need to read in an **RDS** file, you can use the `read_rds()` function.
+
+**Note** that there *is* a function `read.csv` which is available by default in R. It's not quite as nice to use, but you will likely see this function in others' code, so we just want to make sure you're aware of it.
+
+### Writing CSV files into RStudio Cloud
+
+The `readr` family of functions also includes _writing_ functions.
+
+If you want to **write** a **CSV**, **TSV**, or **RDS** file you can use the `write_csv()`, `write_tsv()` or `write_rds()` functions!
```

Comment:
Makes sense to add about the writing files functions after we've talked about the reading files functions.